codegen/src/generateWithOutput.ts

```diff
@@ -1,24 +1,19 @@
 import { Ora } from 'ora';
 import { relative } from 'path';
-import { generate, Language } from './codegen';
-import { getExtensionFromLanguage } from './getExtensionFromLanguage';
+import { generate } from './codegen';
 
 export const generateWithOutput = async ({
-  lang,
   out,
   xataDirectory,
   spinner
 }: {
   spinner: Ora;
   xataDirectory: string;
   out: string;
-  lang: Language;
 }) => {
   spinner.text = 'Found schema, generating...';
 
-  await generate({ xataDirectory, outputFilePath: out, language: lang });
+  await generate({ xataDirectory, outputFilePath: out });
```

Comment:
Nitpick. This function could just have two arguments: xataDirectory and outpuFilePath. Not really necessary to have an object as arguments if only has two fields and both are required and always passed in the same order.

Reply:
You can absolutely ignore this nitpick.

Reply:
I don't really have a strong opinion, but with object syntax, we get some benefits:

1. Autocompletion. If we call `generate()`, TS will tell us we need an object, if we run `generate({})`, TS will tell us `xataDirectory` and `outputFilePath` are missing.
2. We can be more liberal with order of options.
3. It's more explicit in that `xataDirectory` is always any given identifier (like `xataDirectory: anyVariable`).

We lose some of these as function arguments:

1. Order of arguments works, and breaks when we change the signature of this function. This is not a huge deal because we have TypeScript so it's still safe.
2. It can be a bit ambiguous to call `generate(arg1, arg2)` without the explicit key/value syntax.

Not a strong opinion, but let me know if you'd like me to change it and I will. 🙌

Reply:
> Autocompletion

vscode will tell you the name of the arguments and their types. So, there's no difference here. You have to type more if the arguments are encapsulated in an object.

> We can be more liberal with order of options.

Sure, but in this case we have always two arguments and both are required.

> Not a strong opinion, but let me know if you'd like me to change it and I will. 🙌

No need to change it. But I think it's something we should follow in general. If the function receives very few arguments and all/most of them are optional, we should use "regular" arguments.